README.md

```diff
@@ -167,7 +167,65 @@ The code snippet shown below is an example of how this can be done.
                  */
                 bind(DynamicConfiguration.class)
                         .annotatedWith(Names.named("cerberus.config"))
-                        .to(cerberusConfig);
+                        .toInstance(cerberusConfig);
+            } else {
+                logger.info("Property corresponding to the Vault path for secrets not found! "
+                        + "Cerberus not added as Configuration source");
+            }
+        }
+    }
+```
+
+#### Using CerberusConfigurationSource or NamespacedCerberusConfigurationSource in a nonpolling fashion
```

Comment:
We want this example to come first before polling

Reply:
Also, to make it clear that only one of these solutions is needed: polling OR non-polling

Reply:
Updated. See if you like the new structure.